src/containers/Bookings.js

```diff
@@ -27,52 +30,71 @@ export default class Bookings extends Component {
   }
 
   toggleColumn = (e) => {
-    console.log(e.currentTarget.innerHTML);
-    let val = e.currentTarget.innerHTML;
     let isSorted = this.state.sort;
-    console.log(isSorted)
     this.setState({
       sort: !isSorted
     });
-    this.sortColumn(val);
+    this.sortColumn();
+  }
+  
+  selectRowHandler = (e) => {
+    {e.currentTarget.className !== 'row-hover'? 
+      ( e.currentTarget.className = this.state.background,
```

Comment:
I think this is mutating the DOM directly. This is something we want to avoid in React as any changes we make can be accidentally wiped by re-rendering. We should use state and tell React how to render the DOM instead.